pyaedt/modeler/circuits/PrimitivesEmit.py

```diff
@@ -743,6 +743,119 @@ def band_stop_frequency(self, band_node, units=""):
             units = self.units["Frequency"]
         return consts.unit_converter(float(band_node.props["StopFrequency"]), "Freq", "Hz", units)
 
+    def set_band_start_frequency(self, band_node, band_start_freq, units=""):
+        """Set start frequency of a given band.
+
+        Parameters
+        ----------
+        band_node : Instance of the band node
+        band_start_freq : float
+        units : str, optional
+
+        Returns
+        ------
+        None
```

Comment:
Is ``None`` really the default?

Reply:
Just like set_band_power_level(), this method does not return anything.